persistent/Database/Persist/Class/PersistUnique.hs

```diff
@@ -492,10 +493,10 @@ getByValueUniques uniqs =
             Just z -> return $ Just z
 
 -- TODO: expose this to users
-recordName
+_recordName
     :: (PersistEntity record)
     => record -> Text
-recordName = unHaskellName . entityHaskell . entityDef . Just
+_recordName = unHaskellName . entityHaskell . entityDef . Just
```

Comment:
If this were exposed to users it would be from the module that defines `EntityDef` I'd think

Reply:
That would mean it'd go into `Database.Persist.Types.Base`, but `entityDef` is defined in `Database.Persist.Class.PersistEntity`, which depends on `Database.Persist.Types.Base`... Should I just throw it into `Database.Persist.Class.PersistEntity`?

Reply:
:thinking: Yeah that makes the most sense. Thanks :)